imap_processing/cdf/config/imap_mag_l2_variable_attrs.yaml

```
@@ -63,20 +89,29 @@ direction_attrs:
     DICT_KEY: SPASE>Support>SupportQuantity:Orientation
 
 
-# TODO: Add magnitude and range attributes, remove this
-fill:
+magnitude:
   <<: *support_default
-  CATDESC: Compression bit width
-  DEPEND_0: epoch
-  DISPLAY_TYPE: time_series
-  FIELDNAM: Compressed data bit width
-  LABLAXIS: Bit width
+  CATDESC: Magnitude of the magnetic field vector
+  FIELDNAM: Magnetic Field Magnitude
+  LABLAXIS: "|B|"
+  FORMAT: F12.5
+  FILLVAL: -1.0e+31
+  UNITS: nT
+  VALIDMIN: 0
+  VALIDMAX: 1.0e+31
```

We will never get more than a few 10s of nT, maybe 50k when we are in Earth's magnetic field. I think Andriy said they wanted these to be semi-realistic estimates.
```suggestion
  VALIDMAX: 1.0e+5
```

---

Good point!